code/index.html

```diff
@@ -1,30 +1,92 @@
+    <title>[BUNS] — Made to Order</title>
+ 
+ 
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="title">
+      <div class="title-text">
+        <h1>[BUNS]</h1></div>
+    </header>
+
 
     <!-- Signup form -->
+  <main class="main">
+    <div class="form">
+      <form action="https://httpbin.org/anything" method="POST">
+
+        <label class="textArea">Your Name:<br>
+        <input type="text" required name="fullname" placeholder="Full Name"><br></label>
+
+        <label class="textArea">Mail:<br>
+        <input type="email" required name="email" placeholder="Mail"><br></label>
+
+        <label class="textArea">Password: <br>
+        <input type="password" required name="pwd" placeholder="*******"><br></label>
 
+        <label class="textArea">Phone:<br>
+        <input type="text" required name="phone" placeholder="07-........"><br></label>
+
+        <label class="choice">Flavour: <br>
+        <input type="radio" name="buns" value="Cinnamon" checked>Cinnamon<br>
+        <input type="radio" name="buns" value="Cardamom">Cardamom<br>
+        <input type="radio" name="buns" value="Vanilla">Vanilla<br></label>
+
+        <label class="checkBox">Allergies: <br>
+        <input type="checkbox" name="option1" value="lactosFree">Lactos-free<br>
+        <input type="checkbox" name="option2" value="glutenFree">Gluten-free<br></label>
+
+        <label>How many?<br>
+        <input type="number" required id="amount" name="amount" min="1" max="20"><br></label>
+        <input class = "button" type="submit" value="Place Order">
+        
+        <p class="info">*Orders over 20 buns will have to be ordered over phone</p>
+      </form>
+    </div>
+  
     <section class="accordion">
       <!-- Accordion -->
       <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
+      <div class="question" id="section1">Do you use real butter?</div>
+      <div class="answer">
+        <p>Of course, we use real butter for a number of reasons but mainly
+          due to taste and product quality.
+        </p>
+      </div>
+
+      <div class="question" id="section2">Is there vegan options?</div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p>At this point - no. We still haven't found a way to keep the taste level 
+          we desire and are therefore still working on a vegan solution.
+        </p>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
+      <div class="question" id="section3">When do I pick them up?</div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p>You'll get a text to the number you used to order with all the info you'll need.</p>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
+
+    </section> 
+  </main>
+
+  <script src="script.js"></script>
+
+</body>
+
+  <footer>
+    <a href="tel: 0171-123 45"><img src="img/phone.jpg" alt="phone"></a>
```

Comment:
In this case the image doesn't fulfil a purpose for someone that can't see it, it's only here to make the phone link to be pretty. So then you can leave `alt` empty! Then a screen reader would skip to read the image and instead only read the link
```suggestion
    <a href="tel: 0171-123 45"><img src="img/phone.jpg" alt=""></a>
```